src/drivers/dw/ssi-spi.c

```diff
@@ -108,7 +108,7 @@ struct spi_dma_config {
 
 struct spi_reg_list {
 	uint32_t ctrlr0;
-	uint32_t ctrlr1;
+	/* uint32_t ctrlr1; unused for now */
```

Comment:
Why do we keep it if i t's unused? We have git to track 'history', no need to clutter the codebase.

Reply:
I didn't check who wrote the code, so wasn't sure what to do with this one.
@lyakh can you comment on whether we should just remove this, or keep it for future extensions, if any?

Reply:
yes, can remove this one, the struct is anyway not complete by any means

Reply:
removed